test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/TagHelpersWebSite.Employee.Create.Invalid.html

```diff
@@ -69,7 +69,7 @@
 
             <div class="form-group">
                 <div class="col-md-offset-2 col-md-10">
-                    <input class="btn btn-default" type="submit" value="Create" />
+                    <input type="submit" value="Create" class="btn btn-default" />
```

Comment:
Used to be a `TagHelper` and it no longer is so the attribute ordering is no longer affected.